cdhweb/projects/migrations/0017_rename_mezz_models.py

```diff
@@ -0,0 +1,99 @@
+# Generated by Django 2.2.17 on 2021-01-14 16:47
+
+import cdhweb.resources.models
+from django.db import migrations, models
+import django.db.models.deletion
+import mezzanine.core.fields
+import mezzanine.utils.models
+import taggit.managers
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sites', '0002_alter_domain_unique'),
+        ('people', '0021_peoplelandingpage'),
+        ('resources', '0008_userresource_to_personresource'),
+        ('taggit', '0003_taggeditem_add_unique_index'),
+        ('projects', '0016_project_member_to_person'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
If we proceed with this approach, we need to make sure this is a rename and not a create!